lib/realm.dart

```diff
@@ -58,7 +58,7 @@ void initRealm() {
       return true;
     }());
     
-    if (!path.isEmpty && path.endsWith("/")) {
+    if (path.isNotEmpty && path.endsWith("/")) {
```

Comment:
Consider using [path](https://pub.dev/packages/path) package for path manipulation.

Reply:
thanks this is old code when we did not depend on path package. Will try to update

Reply:
actually since this is split by platform I don't see the gain of using path here. path pkg is useful when dealing with paths on generic code. but here we are explicit for every single platform.